### PR TITLE
Explain plugin refs in overview

### PR DESCRIPTION
**Pull request by KnorpelSenf**
This was done in #950 but forgotten in #1008 that partially superseded #950.

### CODE REVIEW

**Review by roj1512 — APPROVED** [review]

File: site/docs/uk/guide/README.md [hunk 1]
```diff
@@ -27,12 +27,14 @@ grammY --- це фреймворк для створення ботів Telegram
 **Перша частина** (ви дивитеся її!) пояснює, як працюють боти та як використовувати grammY.
 Це те, чим ви будете користуватися найчастіше.
 
-Розділ _Вивчення_ завжди є хорошим початком.
+Розділ _Вивчення_ завжди є хорошим початком, оскільки в ньому пояснюються всі основні концепції.
 Також перегляньте нашу чудову колекцію _Плагінів_ і подивіться на _Приклади_.
 
-**Друга частина** --- це [довідка API grammY](/ref/core/), посилання на яке знаходиться у верхній частині сторінки.
+**Друга частина** --- це [довідка API grammY](/ref/), посилання на яке знаходиться у верхній частині сторінки.
```
Comment (LWJerri) [inline site/docs/uk/guide/README.md:33]:
```suggestion
**Друга частина** --- це [довідка API grammY](/ref/), яка знаходиться у верхній частині сторінки.
```

File: site/docs/uk/guide/README.md [hunk 2]
```diff
@@ -27,12 +27,14 @@ grammY --- це фреймворк для створення ботів Telegram
...
 Це детальний огляд кожного окремого біта коду, який надає grammY.
 Він автоматично генерується з коду grammY і містить усі корисні пояснення, які зазвичай можна знайти, навівши курсор на будь-який елемент grammY у редакторі коду.
+Те саме стосується кожного плагіна в екосистемі grammY.
+Окрім того, що плагіни зʼявляються в [огляді довідки API](/ref/), посилання на їхню довідку API містяться в короткому описі внизу сторінки кожного плагіна.
```
Comment (LWJerri) [inline site/docs/uk/guide/README.md:37]:
```suggestion
Окрім того, що плагіни зʼявляються в [огляді довідки API](/ref/), посилання на довідку містяться в короткому описі внизу сторінки кожного плагіна.
```

**Review by LWJerri — APPROVED** [review]
You can accept or reject these suggestions, they don't make much difference, but readability seems to me to be getting better. :shipit:

File: site/docs/id/guide/README.md [hunk 3]
```diff
@@ -26,13 +26,15 @@ Dokumentasi bot grammY dibagi menjadi tiga area:
 
 **Area pertama**---yang sedang kamu baca saat ini---berisi tutorial penggunaan grammY dan penjelasan mengenai cara kerja dari suatu bot.
 Dokumen ini yang akan paling sering kamu gunakan.
-Kamu bisa memulai dari bab _Belajar_ di tab navigasi yang ada di bagian atas halaman ini.
+Kamu bisa memulai dari bab _Belajar_ di tab navigasi yang ada di bagian atas halaman ini, disana dijelaskan semua konsep intinya.
```
Comment (quadratz) [inline site/docs/id/guide/README.md:29]:
```suggestion
Untuk memahami konsep dasar secara keseluruhan, kamu bisa memulai dari bab _Belajar_ yang berada di tab navigasi bagian atas halaman ini.
```

File: site/docs/id/guide/README.md [hunk 4]
```diff
@@ -26,13 +26,15 @@ Dokumentasi bot grammY dibagi menjadi tiga area:
...
 Jangan lupa untuk melihat koleksi _Plugin_ kami yang menakjubkan serta berbagai _Contoh_ yang sudah disediakan.
 
-**Area kedua** adalah [referensi API grammY](/ref/core/), yang juga bisa diakses melalui tab navigasi di bagian atas halaman.
+**Area kedua** adalah [referensi API grammY](/ref/), yang juga bisa diakses melalui tab navigasi di bagian atas halaman.
 Referensi ini berisi gambaran terperinci dari setiap bit kode yang disediakan grammY.
 Ia juga menghasilkan penjelasan-penjelasan bermanfaat dalam bentuk tooltip yang dibuat secara otomatis dari kode program grammY.
 Tooltip tersebut dapat ditemukan dengan mengarahkan kursor ke berbagai elemen grammY di kode editor.
+Hal yang sama juga berlaku pada setiap plugin pada ekosistem grammY.
```
Comment (quadratz) [inline site/docs/id/guide/README.md:36]: Pengulangan `pada`
```suggestion
Hal yang sama juga berlaku untuk setiap plugin di ekosistem grammY.
```

File: site/docs/id/guide/README.md [hunk 5]
```diff
@@ -26,13 +26,15 @@ Dokumentasi bot grammY dibagi menjadi tiga area:
...
+Selain menampilkan [gambaran referensi API](/ref/), tiap plugin juga memiliki referensi API-nya sendiri yang tertaut di ringkasan plugin pada bagian bawah halaman.
```
Comment (quadratz) [inline site/docs/id/guide/README.md:37]:
```suggestion
Selain menampilkan [gambaran referensi API](/ref/), setiap plugin juga memiliki referensi API-nya sendiri yang tertaut di ringkasan plugin pada bagian bawah halaman.
```

**Review by quadratz — CHANGES_REQUESTED** [review]

File: site/docs/id/guide/README.md [hunk 6]
```diff
@@ -26,13 +26,15 @@ Dokumentasi bot grammY dibagi menjadi tiga area:
...
+Untuk memahami konsep dasar secara keseluruhan, kamu bisa memulai dari bab _Belajar_ yang berada di tab navigasi bagian atas halaman ini.
 Jangan lupa untuk melihat koleksi _Plugin_ kami yang menakjubkan serta berbagai _Contoh_ yang sudah disediakan.
 
-**Area kedua** adalah [referensi API grammY](/ref/core/), yang juga bisa diakses melalui tab navigasi di bagian atas halaman.
+**Area kedua** adalah [referensi API grammY](/ref/), yang juga bisa diakses melalui tab navigasi di bagian atas halaman.
 Referensi ini berisi gambaran terperinci dari setiap bit kode yang disediakan grammY.
 Ia juga menghasilkan penjelasan-penjelasan bermanfaat dalam bentuk tooltip yang dibuat secara otomatis dari kode program grammY.
 Tooltip tersebut dapat ditemukan dengan mengarahkan kursor ke berbagai elemen grammY di kode editor.
+Hal yang sama juga berlaku untuk setiap plugin di ekosistem grammY.
+Selain menampilkan [gambaran referensi API](/ref/), setiap plugin juga memiliki referensi API-nya sendiri yang tertaut di ringkasan plugin pada bagian bawah halaman.
```
Comment (quadratz) [inline site/docs/id/guide/README.md:37]: > In addition to appearing in the [overview of API references](/ref/), plugins have their API reference linked in the plugin summary at the bottom of each plugin page.

Based on that context, I think this one is more appropriate

```suggestion
Selain tercantum di [referensi utama API](/ref/), setiap plugin juga mencantumkan referensi API-nya masing-masing di ringkasan plugin pada bagian bawah halaman.
```

**Review by quadratz — CHANGES_REQUESTED** [review]
@dzikrl

File: site/docs/uk/guide/README.md (hunk 2 above)
Comment (deptyped) [inline site/docs/uk/guide/README.md:37]:
```suggestion
Окрім того, що плагіни представлені в [огляді довідки API](/ref/), посилання на їхню довідку містяться в короткому описі внизу сторінки кожного плагіна.
```

File: site/docs/uk/guide/README.md (hunk 1 above)
Comment (deptyped) [reply site/docs/uk/guide/README.md:33]:
```suggestion
**Друга частина** --- це [довідка API grammY](/ref/), посилання на яку знаходиться у верхній частині сторінки.
```

File: site/docs/uk/guide/README.md (hunk 2 above)
Comment (deptyped) [reply site/docs/uk/guide/README.md:37]:
```suggestion
Окрім того, що плагіни представлені в [огляді довідки API](/ref/), посилання на їхню довідку містяться в короткому описі внизу сторінки кожного плагіна.
```

**Review by deptyped — APPROVED** [review]

**Pull request merged**